doc/rfcs/003-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -0,0 +1,172 @@
+
+![Mockup: Remove Past Collaborator](https://user-images.githubusercontent.com/378023/37639861-67778f8e-2c56-11e8-994f-428802fa9191.png)
+
+Once you've removed someone from your list of past collaborators, you will appear as "offline" in their list of past collaborators, and they can no longer invite you to their portal from directly within Atom.
+
+If you later decide that you want to collaborate with them again, you can send them a URL (via Slack, IRC, etc.) to join your portal, or you can ask them to send you a URL (via Slack, IRC, etc.) so that you can join their portal. Once you collaborate in a portal together again, Teletype adds them back to your list of past collaborators, and they can once again see your online/offline status in their list of past collaborators.
+
+### Mockups
+
+#### Inviting a past collaborator
+
+![](https://user-images.githubusercontent.com/378023/37643605-6947af84-2c64-11e8-94b6-849e81fd8cd0.png)
+
+### Collaborator states
+
+You can see the collaborators that are currently in your portal, the state of the other invitations that you've sent, and the online/offline state of all past and current collaborators:
+
+![](https://user-images.githubusercontent.com/378023/37642043-1d66797e-2c5f-11e8-8def-01f063888979.png)
+
+The list of past collaborators has a `max-height`. If the list exceeds the `max-height`, you can scroll. The scrolling area includes sent and rejected invitations.
+
+#### Simultaneously participating in multiple portals
+
+While probably not very common, it's possible to host a portal while also participating as a guest in other portals.
+
+![](https://user-images.githubusercontent.com/378023/37640529-6a5ddaac-2c59-11e8-82f5-9ee4c2ed42a5.png)
+
+## Out of scope
+
+In the interest of getting the highest impact functionality in users' hands as quickly as possible and then iterating based on real-world feedback, the following functionality is out of scope for this RFC, but may be addressed in follow-up releases.
+
+- Changing your online/offline status
+    - Setting your status to offline. (In the meantime, you can sign out of Teletype or disable Teletype in Atom's package settings when you want to use Atom while not accepting invitations from your past collaborators.)
+    - Setting your status to "away" or "busy"
+- UX enhancements to the list of past collaborators
+    - Changing the sort order for your list of past collaborators (e.g., sort by how recently you've collaborated, sort or group by online/offline status)
+    - Limiting the size of your list of past collaborators. (In the meantime, you can remove past collaborators to reduce the size of the list.)
+    - Filtering your list of past collaborators
+    - Marking certain collaborators as "favorites"
+- Offering to install Teletype if you click on a portal URL and don't yet have Teletype installed ([#220](https://github.com/atom/teletype/issues/220))
+- Selecting a past collaborator and asking to join their portal
```

Comment:
I am concerned about this potential feature. If they have removed you as a past collaborator, then is this feature overriding their preference?